README.md

```diff
@@ -6,9 +6,39 @@ Be advised: This plugin currently migrates the `RestApi` resource to a nested st
 
 The goal if this plugin is to split some resources off in to nested stacks to work around the 200 CloudFormation resource limit.
 
-Migrating resources to nested stacks is tricky beacuse some plugins rely on querying the resource from the main stack and would need to understand this. There are also plenty of issues with moving resources in existing deployments (you frequently get 'resource already exists' errors). Because of this, this plugin is very conservative. It moves only resources of types that seem to be easy to move.
+Migrating resources to nested stacks is tricky because some plugins rely on querying the resource from the main stack and would need to understand this. There are also plenty of issues with moving resources in existing deployments (you frequently get 'resource already exists' errors). Because of this, this plugin is very conservative. It moves only resources of types that seem to be easy to move.
 
-You can see a list of migrations in `lib/migrate-resources.js`.
+## Stack mappings
+
+__Default migrations map is configured within [stacks-map.js](https://github.com/dougmoscrop/serverless-plugin-split-stacks/blob/master/stacks-map.js)__
+
+
+This map can be customized. To do so, introduce the `stacks-map.js` module in a root folder of your project (this module if exists will be transparently loaded by the plugin).
+
+Example of customization, that moves DynamoDB resources to nested stack:
+
+```javascript
+const stacksMap = require('serverless-plugin-split-stacks/stacks-map');
```

Comment:
I think this is good to merge, the last thing I can think of is this as a design question: do you want to expose .stacks-map as a filesystem interface, or could it instead be a (static?) property of the plugin. It boils down to the alternative being:

`stacksMap = require('serverless-plugin-split-stacks').stacksMap`

Reply:
Good call. I've updated it. 

I've also changed class name into `ServerlessPluginSplitStack`, this name is used by Serverless in eventual error reporting, it's good to have it reflect plugin name directly.